File: CursoPOOUber/Python/UberVan.py
#!/usr/bin/python3


from car import Car

class UberVan(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, licence, driver, typeCarAccepted, seatsMaterial):
        super().__init__(licence, driver)
        self.seatsMaterial = seatsMaterial
        self.typeCarAccepted = typeCarAccepted

File: CursoPOOUber/Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;


class UberVan extends Car {

    Map<String, Map<String, Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberVan(String licence, Account driver) {
        super(licence, driver);
    }
}
